README.md

```diff
@@ -2,7 +2,7 @@
 
 ## react-clipboard-button
 
-- [x] very light (~33KB not minified)
+- [x] very light (~1.2KB not minified)
```

Comment:
by extenalizing the `clipboard` dependency in vite config.

Reply:
Hey @RobinTail , thanks for the contribution!
Regarding this change,  if I'm not mistaken, it means users will have to `npm install clipboard` in addition to installing our library ? Any reason why you are removing the dependency from the bundle ?

Reply:
no, it will happen automatically, because `clipboard` is listed as the production dependency, @guipas 
Therefore it does not have to be bundled, because it's installed along with your packaage.

Reply:
https://github.com/guipas/react-clipboard-button/blob/8e057202d4518c6655417b5fba50a7a581cb3a8d/package.json#L28-L29

Listing in `dependencies` makes clipboard installed into `node_modules` on the consumer side along with this package. It's a subdependency accessible during the build on the consumer side. So you don't need to bundle `clipboard` into `react-clipboard-button`.

Reply:
ok yeah that makes sense! thank you, I'll merge this PR

Reply:
I'll publish on npm later this week